### PR TITLE
Genai update stack core sec sync

### PR DESCRIPTION
**Pull request by daniel-butler-irl**
### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by rajatagarwal-ibm — DISMISSED** [review]

File: ibm_catalog.json
Comment (ocofaigh) [inline ibm_catalog.json:19]: should should be `IBM`

File: ibm_catalog.json
Comment (ocofaigh) [inline ibm_catalog.json:21]: suggest to use same as this: https://github.com/terraform-ibm-modules/stack-ibm-core-security-services/blob/b2ba1d88324f3063cb706f81f394920093929e06/ibm_catalog.json#L34

File: stack_definition.json
Comment (ocofaigh) [inline stack_definition.json:183]: hmm, should this have `Security Service` in the name? Its not really one of the security services (although it is a member of the core security services stack)

File: stack_definition.json
Comment (ocofaigh) [inline stack_definition.json:271]: same as other comment, this is not a "Security Service"

**Review by ocofaigh — CHANGES_REQUESTED** [review]
Set default value of `enable_platform_logs_metrics` to true, and `secret_manager_service_plan` to `standard` since this tile will be going into prod catalog (same as core security services). For the tests you can keep these as `false` and `trial` though

File: ibm_catalog.json
Comment (daniel-butler-irl) [reply ibm_catalog.json:19]: Majority of catalog json is copy pated from the watson da to prevent overriding

File: stack_definition.json
Comment (daniel-butler-irl) [reply stack_definition.json:183]: This is the naming format from the rag stack...

File: ibm_catalog.json
Comment (daniel-butler-irl) [reply ibm_catalog.json:21]: same as above comes from Watson SaaS

File: stack_definition.json
Comment (daniel-butler-irl) [reply stack_definition.json:271]: The is the same as above to indicate it is part of security services not really anything to do with genai

File: ibm_catalog.json
Comment (ocofaigh) [reply ibm_catalog.json:19]: then it needs to be updated in theirs :) - can we reach out?

File: ibm_catalog.json
Comment (ocofaigh) [reply ibm_catalog.json:21]: There not going to be able to GA with that - can you reach out?

File: stack_definition.json
Comment (ocofaigh) [reply stack_definition.json:183]: right, but not what we used in core security services - we should run by team (Allen) for final review on member names

**Review by ocofaigh — APPROVED** [review]
I'm approving so it wont hold up merge - but we need to follow up on some of the items

**Pull request merged**
